stix_shifter_modules/stix_bundle/stix_transmission/connector.py

```diff
@@ -39,17 +46,12 @@ def match(self, pattern, observed_data_sdos, verbose=False):
     def ping_connection(self):
         return_obj = dict()
 
-        if not self.bundle_url:
-            self.bundle_url = self.connection.get('host')
-        auth = self.configuration.get('auth')
-
-        response = self.call_api(self.bundle_url, auth, 'head')
+        response = self.client.call_api(self.bundle_url, 'head', timeout=self.timeout)
         response_txt = response.raise_for_status()
-        response_code = response.status_code
 
-        if response_code == 200:
+        if response.code == 200:
             return_obj['success'] = True
-        elif response_code == 301:
+        elif response.code == 301:
```

Comment:
I get error when response code is 301: `ResponseWrapper' object has no attribute 'headers'`

Redirection occurs when the url contains `raw.github.com` domain instead of `raw.githubusercontent.com`

I think you just need to change line 55 to `response.response.headers.get('Location')`

Reply:
nice catch, fixed